content/en/community/end-user/_index.md

```diff
@@ -5,78 +5,33 @@ Description: Vendor-agnostic resources for users of OpenTelemetry
 ---
 
 The [OpenTelemetry End User Working Group][] has heard feedback from users who
-desire a vendor-agnostic space to discuss adopting OpenTelemetry, so we’re
-excited to announce two new ways for you to connect with other users of the
-project and share best practices, a synchronous monthly discussion group and a
-new private slack channel. These forums will bring together operations and
+desire a vendor-agnostic space to discuss adopting OpenTelemetry, so we provide
+multiple ways for you to connect with other users of the project and share best
+practices:
+
+* [A synchronous monthly discussion group](/community/end-user/discussion-group/)
+* [A private slack channel](/community/end-user/slack-channel/)
+* [Talks about OTel in practice](/community/end-user/otel-in-practice/)
```

Comment:
Would be good for these to be links, eg to the Slack channel itself.

Reply:
it's a private slack channel, so providing a link doesn't do the trick.